eos-tech-support/eos-diagnostics

```diff
@@ -558,6 +558,11 @@ let diagnostics = [
         hardwareInfo: true,
         content: function() { return trySpawn('journalctl --no-pager --output verbose'); },
     },
+    {
+        title: 'Coredump',
```

Comment:
I think it is misleading to have this section if coredumps are not enabled.  Ideally, we would only include this section at all if coredumps are enabled.  So, an empty coredump list would actually mean that the core was not dumped.  Or, perhaps even better, always have the section but explicitly identify whether or not coredumps are enabled.